test/Microsoft.AspNetCore.Rewrite.Tests/UrlMatches/IntegerMatchTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Rewrite;
+using Microsoft.AspNetCore.Rewrite.Internal;
+using Microsoft.AspNetCore.Rewrite.Internal.UrlActions;
+using Microsoft.AspNetCore.Rewrite.Internal.UrlMatches;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Rewrite.Tests.UrlMatches
+{
+    public class IntegerMatchTests
```

Comment:
Other tests that might be good to add:
"0", string.Empty, integers to big for the range of `System.Int32`